### DISCUSSION TITLE
Imprimir a partir de um aplicativo

### DISCUSSION THREAD

**Discussion by ebapp-dev** [Flutterando/forum#136] [Perguntas e Respostas]
Tenho uma necessidade atual de imprimir uma etiqueta a partir de um aplicativo. Estou conseguinda fazer isso através de uma pequena impressora bluetooth, mas essa é uma solução provisória, porque a impressora não é ideal para etiquetas. Preciso de uma solução que imprima etiquetas em uma impressora bluetooth como Zebra ou Brothe, através de bluetooth ou outra solução limpa sem necessidade de outros muitos recursos. Alguém já teve experiência para comártilhar sobe isso?

**Comment by EduardoM-GO** [comment]
Talvez esse package pode te ajudar: [flutter_pos_printer_platform](https://pub.dev/packages/flutter_pos_printer_platform)
